service/handlers.go

```diff
@@ -90,6 +90,9 @@ func TokenHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
 	w.Header().Set("X-CSRF-Token", csrf.Token(r))
 
+	// Check the JWT and return it in the authorization header, if valid
+	JWTCheck(w, r)
```

Comment:
No - this is just used by the UI. So the JWT will be set correctly or not. If not, the UI will set up nil permissions.